### PR TITLE
Portfolio details popup window"

### PR DESCRIPTION
**Pull request by Hamayounali**
1. Updated index.html file 
2. Updated styles.css file 
3. Updated script.js file 
4. Created popup.js file

### CODE REVIEW

File: assets/js/popup.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const arr2 = [
+  'Data Dashboard Health Care',
+  'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard',
+  'HTML',
+  'BootStrap',
+  'Ruby',
+  'See Live',
+  'See Source',
+  'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry Ipsum is simply dummy text of the printing and typesetting industry.',
+];
```
Comment (ShahierNashaat) [inline assets/js/popup.js]: - [ ] Kindly be noticed that the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featured image, technologies, link to the live version, and link to the project source code. Ie:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

File: index.html [hunk 2]
```diff
@@ -158,156 +161,70 @@ <h1 class="work-sample-2"> Multy-Post Stories </h1>
...
+    	<span class="close">&times;</span>
+
+    	<div class="popup-flex-container">
+    		<div class="div-shadow-1" id="div-container">
+    		</div>
+    		<div class="div-shadow-2" id="div-container2">
+    		</div>
+    	</div>
+    	
+    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please follow the Figma design in your project popup window as you can see in the Figma design there is extra space at the bottom of the mobile popup window.

Your Design | Figma Design
---------------- | --------------------
Screenshot from 2022-06-24 16-11-44 | Screenshot from 2022-06-24 16-11-23

File: assets/js/popup.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  img2.src = 'assets/images/source2.png';
+  img2.classList.add('tag-popup-2');
+  DivContainer3.appendChild(img2);
+}
+
+function myFunction() {
+  popup.style.display = 'block';
+  popupContent();
+  popupContent2();
+}
```
Comment (ShahierNashaat) [inline assets/js/popup.js]: - [ ] Kindly be noticed that when the `See Project` button is clicked you should display the right project according to the button ie: if the `See Project` is clicked for `Tonic` project it should display the data for `Tonic` project.

File: index.html [hunk 4]
```diff
@@ -158,156 +161,70 @@ <h1 class="work-sample-2"> Multy-Post Stories </h1>
...
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-11">
+			<button class="see" type="button" onclick="myFunction(this)" id="9">See Project</button>
 		</div>
 	</div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the `See Project` button for some projects is not visible please fix that.

Screenshot from 2022-06-24 16-27-47

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Hamayounali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Hamayounali 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**